README.md

```diff
@@ -132,7 +132,23 @@ To help you understand how Olares stands out in the landscape, we've created a c
 
 ## Getting started
 
-Refer to [Getting Started Guide](https://docs.olares.xyz/manual/get-started/) to spin up your Olares on Linux, Windows, Mac, or Raspberry Pi.
+### System compatibility
+Olares is available for Linux, Raspberry Pi, Mac, and Windows. It has been tested and verified on the following systems:
+
+| Platform            | Operating system                     | Notes                                                 |
+|---------------------|--------------------------------------|-------------------------------------------------------|
+| Linux               | Ubuntu 24.04 <br/> Debian 12.8       |                                                       |
+| Raspberry Pi        | RaspbianOS                           | Verified on Raspberry Pi 4 Model B and Raspberry Pi 5 |
+| Windows             | Windows 11 23H2 <br/>Windows 10 22H2 |                                                       |
+| Mac (Apple silicon) | macOS Ventura 13.3.1               |                                                       |
+| Proxmox VE (PVE)    | Proxmox Virtual Environment 8.0      |                                                       |
+
+> **Note**
+> 
+> If you successfully install Olares on an operating system that is not listed in the compatibility table, please let us know! You can [open an issue](https://github.com/beclab/Olares/issues/new) or submit a pull request on our GitHub repository.
+
+### Set up Olares
+To get started with Olares on your own device, follow the [Getting Started Guide](https://docs.olares.xyz/manual/get-started/) for step-by-step instructions.
 
```

Comment:
Made some changes to improve readbility and clarity:
- Group operating systems under the same platform to avoid repetition
- Remove the "Installation support" column because all os listed here are tested and supported
- Add tested models for raspberry pi in "Notes" column
- Add a note to call for community feedback
- Use level 3 headings to separate compatibility table and reference to doc

Reply:
Should we include architecture in this table too?